### PR TITLE
[FX-1752] Add "Consign" to desktop global nav More menu; nav code cleanup

### PR DESCRIPTION
**Pull request by anandaroop**
https://artsyproduct.atlassian.net/browse/FX-1752

This adds "Consign" to the global nav, and cleans up some stale/unused code.

more

---

- The main change here is the tiny commit in 3f2bb7f

- In  493908b I also took this opportunity to delete the experimental version of the nav bar from the A/B test a few weeks ago.  (FX decided to deprecate this code to reduce ambiguity; we can re-create it as needed if/when we re-open the test. cc: @damassi)

- I also found that there were two versions of the mobile dropdown menu —  a [stale version](https://github.com/artsy/reaction/blob/dd6869df3794df9304323b52ca3a8fa3f49c4903/src/Components/Nav/MobileMenu.tsx), and a [current version](https://github.com/artsy/reaction/blob/master/src/Components/NavBar/Menus/MobileNavMenu.tsx). I remove the stale one in 872a5c8. (This entailed deleting a publishing component which depended on it as well. But as best as I can tell that publishing [`SubNav` component](https://github.com/artsy/reaction/blob/dd6869df3794df9304323b52ca3a8fa3f49c4903/src/Components/Publishing/Nav/SubNav.tsx) is no longer used either. cc: @eessex)

### CODE REVIEW

File: src/Components/Nav/MobileMenu.tsx [hunk 1]
```diff
@@ -1,61 +0,0 @@
...
-  background-color: black;
-  left: 0;
-  padding: 20px;
-  margin-top: ${props => props.navHeight};
-  ${StyledTextLink} {
-    display: inline-block;
-    width: 50%;
-    line-height: 2;
-  }
-`
```
Comment (damassi) [inline src/Components/Nav/MobileMenu.tsx]: Where'd the mobile nav menu go?

Comment (damassi) [reply src/Components/Nav/MobileMenu.tsx]: Ahh i see -- unrelated to navbar rebuild. Yeah, we gotta clean up that folder...

File: src/Components/NavBar/Menus/MoreNavMenu.tsx [hunk 2]
```diff
@@ -33,6 +33,7 @@ export const MoreNavMenu: React.FC<{ width?: number }> = ({ width = 160 }) => {
       <MenuItem href="/artists">Artists</MenuItem>
       <MenuItem href="/shows">Shows</MenuItem>
       <MenuItem href="/institutions">Museums</MenuItem>
+      <MenuItem href="/consign">Consign</MenuItem>
```
Comment (damassi) [inline src/Components/NavBar/Menus/MoreNavMenu.tsx]: Should this be added on mobile as well?

File: src/Components/Nav/MobileMenu.tsx (hunk 1 above)
Comment (anandaroop) [reply src/Components/Nav/MobileMenu.tsx]: Yeah totally. I was just trying to do my part for this little corner of Reaction 👼

File: src/Components/NavBar/Menus/MoreNavMenu.tsx (hunk 2 above)
Comment (anandaroop) [reply src/Components/NavBar/Menus/MoreNavMenu.tsx]: Not as part of this ticket at least.

(Not all of the desktop CTAs  — join as a gallery, consign your works — are in the mobile nav. But I'll ping folks to be sure. Can follow up in another PR if necessary.)

File: src/Components/Nav/MobileMenu.tsx (hunk 1 above)
Comment (damassi) [reply src/Components/Nav/MobileMenu.tsx]: tyty!

File: src/Components/Publishing/Nav/SubNav.tsx [hunk 3]
```diff
@@ -1,86 +0,0 @@
-import { avantgarde } from "Assets/Fonts"
-import React from "react"
```
Comment (eessex) [inline src/Components/Publishing/Nav/SubNav.tsx]: This file is used in Force

**Comment by xtina-starr** [comment]
This looks good to go.

**Comment by artsyit** [comment]
:rocket: PR was released in `v25.13.0` :rocket:

**Pull request merged**